download_and_create_reference_datasets/v03/write_combined_interval_reference_ht.py

```diff
@@ -49,16 +24,26 @@ def run(environment: str, dataset: str):
         genome_version=genome_version,
     )
     if hl.hadoop_exists(os.path.join(destination_path, '_SUCCESS')):
-        ht = update_existing(destination_path, dataset, genome_version)
+        ht = update_existing_joined_hts(
+            destination_path,
+            dataset,
+            DATASETS,
+            VERSION,
+            genome_version,
+        )
     else:
-        ht = create_new(genome_version)
+        ht = join_hts(DATASETS, VERSION, reference_genome=genome_version)
```

Comment:
I know we have tests but I think keeping a ht.describe() is helpful when running this just so the user can visually see the table is what they expect.